code/modules/shuttle/emergency.dm

```diff
@@ -221,6 +240,44 @@
 
 	priority_announce("The emergency shuttle has been called. [redAlert ? "Red Alert state confirmed: Dispatching priority shuttle. " : "" ]It will arrive in [timeLeft(600)] minutes.[reason][SSshuttle.emergencyLastCallLoc ? "\n\nCall signal traced. Results can be viewed on any communications console." : "" ]", null, 'sound/AI/shuttlecalled.ogg', "Priority")
 
+/obj/docking_port/mobile/emergency/proc/legitimize_call() //Checks are sorted in rough order of processing cost
+	if(SSshuttle.force_shuttle) //Set manually by certain "Oh shit" events, like narsie.
+		message_admins("Shuttle will arrive due to you know what.")
```

Comment:
Why not have force_shuttle be a string for the reason. So it can be "summoned Ratvar", "summoned Narsie", "admin fiat".